### ISSUE TITLE
[Enhancement]: Update price markup in widget Recently Viewed Products

### ISSUE THREAD

**Issue by khoipro** [woothemes/woocommerce#45263]
### Describe the solution you'd like

When implementing a Recently Viewed Products widget, I saw a markup:

```php
// templates/content-widget-product.php
<a href="">
  <img src="">
  <span class="product-title"></span>
</a>
<del></del>
<ins></ins>
```

The product price markup does not match what we expected (missing wrapper span.price outside)

```
<span class="price">
  <del></del>
  <ins></ins>
</span>
```

Also from what I expected, a price block should be inside link.

### Describe alternatives you've considered

```php
<a href="<?php echo esc_url( $product->get_permalink() ); ?>">
  <?php echo $product->get_image(); // PHPCS:Ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
  <span class="product-title"><?php echo wp_kses_post( $product->get_name() ); ?></span>
  <span class="price"><?php echo $product->get_price_html(); ?></span>
</a>
```

### Additional context

I can myself deploy that pull request if needed.

**Comment by khoipro** [comment]
I've attached a markup comparing price block in widget and in default loop has no same style (missing span.price).

![image](https://github.com/woocommerce/woocommerce/assets/10395311/7c0aecc0-61a2-44b4-88de-28445359c6c4)

**Comment by roykho** [comment]
Hi @khoipro - thanks for opening the issue. Can you please clarify if you're using a non blocks based theme (since you mentioned "widget").

**Comment by khoipro** [comment]
Yes, this is not a block widget. We use classic widgets @roykho in this case.

**Comment by dinhtungdu** [comment]
@khoipro IMO, the best option here is to override the template `content-widget-product.php` with your version. This file has been there since 2013, and many sites rely on the current markup. Any change here could lead to a broken layout issue for sites using the default template.

**Comment by gigitux** [comment]
Thanks for opening this issue!

As @dinhtungdu suggested, the best option is to override the `content-widget-product.php` template.

While we appreciate you sharing your idea, it doesn’t align with our current project priorities. That said, this doesn’t mean it’s not a good suggestion — just that our resources are limited at the moment.

We may revisit this in the future as priorities shift, but in the meantime, please don’t be alarmed if there’s no immediate activity on this issue.
